package.json

```diff
@@ -149,4 +149,4 @@
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
   }
-}
+}
```

Comment:
Annoying change but can you add a new line at the end?

Reply:
sure